components/central-application-connectivity-validator/README.md

```diff
@@ -0,0 +1,73 @@
+# Central Application Connectivity Validator
+
+## Overview
+
+The Central Application Connectivity Validator validates client certificate subjects.
+It proxies the requests to the Event Service and the Application Registry.
+
+The Central Application Connectivity Validator has the following parameters:
+- **proxyPort** is the port on which the reverse proxy is exposed. The default port is `8081`.
+- **externalAPIPort** is the port on which the external API is exposed. The default port is `8080`.
+- **tenant** the name of the tenant (subject field `OrganizationalUnit`) for which client certificate should be generated. When empty, the tenant validation is skipped.
+- **group** the name of the group (subject field `Organization`) for which client certificate should be generated. When empty, the group validation is skipped.
+- **eventServicePathPrefixV1** is the path prefix for which requests are forwarded to the Event Service V1 API. The default value is `/%%APP_NAME%%/v1/events`.
+- **eventServicePathPrefixV2** is the path prefix for which requests are forwarded to the Event Service V2 API. The default value is `/%%APP_NAME%%/v2/events`.
+- **eventMeshHost** is the host and the port of the Event Mesh adapter. The default value is `eventing-event-publisher-proxy.kyma-system`.
+- **eventMeshDestinationPath** is the destination path for the requests coming to the Event Mesh. The default value is `/publish`.
+- **appRegistryPathPrefix** is the path prefix for which requests are forwarded to the Application Registry. The default value is `/%%APP_NAME%%/v1/metadata`.
+- **appRegistryHost** is the host and the port of the Event Service. The default value is `application-registry-external-api:8081`.
+- **appNamePlaceholder**  is the path URL placeholder used for an application name. The default value is `%%APP_NAME%%`.
```

Comment:
%%APP_NAME%% is really used in the application, so the documentation provides this value. The "{}" can be used by some template engines, so that's whay '%%' was chosen

Reply:
OK, I've updated the comment accordingly.